src/plugins/nvme/nvme.h

```diff
@@ -35,8 +37,9 @@ typedef enum {
 
 typedef enum {
     BD_NVME_TECH_MODE_INFO         = 1 << 0,
-    BD_NVME_TECH_MODE_NAMESPACE    = 1 << 1,
-    BD_NVME_TECH_MODE_INITIATOR    = 1 << 2,
+    BD_NVME_TECH_MODE_OP           = 1 << 1,
```

Comment:
The new mode is missing from the API file. I also think a little more descriptive name than `OP` could be useful.

Reply:
Hmm, I was also thinking about dropping this categorization. Since this is a first commit and all the functionality will be included unconditionaly, it would only serve an information purpose.

Reply:
Renamed to `BD_NVME_TECH_MODE_MANAGE`